.mergify.yml

```diff
@@ -11,6 +11,23 @@ queue_rules:
       {{ body }}
     merge_method: squash
 pull_request_rules:
+  - name: auto-add v4.x backport label when v3.x label is added
+    conditions:
+      - base=main
+      - label=backport-to-release/v3.x
+      - -label=backport-to-release/v4.x
```

Comment:
actually, this condition should be removed, no?
Cause we want to add the `backport-to-release/v4.x` label when there's a PR with these conditions:

```
 - base=main
 - label=backport-to-release/v3.x
 ```